miio/cooker_multi.py

```diff
@@ -0,0 +1,419 @@
+    def __init__(self, data):
+        self.data = data
+
+    @property
+    def mode(self) -> OperationMode:
+        """Current operation mode."""
+        return OperationMode(self.data["status"])
+
+    @property
+    def menu(self) -> str:
+        """Selected menu id."""
+        try:
+            return COOKING_MENUS[self.data["menu"]]
+        except KeyError:
+            return "Unknown menu"
+
+    @property
+    def stage(self) -> str:
+        """Current stage if cooking."""
+        try:
+            return COOKING_STAGES[self.data["phase"]]["name"]
+        except KeyError:
+            return "Unknown stage"
+
+    @property
+    def temperature(self) -> Optional[int]:
+        """Current temperature, if idle.
+
+        Example values: 29
+        """
+        return self.data["temp"]
+
+    @property
+    def start_time(self) -> int:
+        """Start time of cooking?"""
+        return int(self.data["t_start"])
+
+    @property
+    def remaining(self) -> int:
+        """Remaining minutes of the cooking process."""
+        return int(int(self.data["t_left"]) / 60)
+
+    @property
+    def cooking_delayed(self) -> Optional[int]:
+        """Wait n minutes before cooking / scheduled cooking."""
+        delay = int(self.data["t_pre"])
+
+        if delay >= 0:
+            return delay
+
+        return None
+
+    @property
+    def duration(self) -> int:
+        """Duration of the cooking process."""
+        return int(self.data["t_cook"])
+
+    @property
+    def keep_warm(self) -> bool:
+        """Keep warm after cooking?"""
+        return self.data["akw"] == 1
+
+    @property
+    def settings(self) -> None:
+        """Settings of the cooker."""
+        return None
+
+    @property
+    def hardware_version(self) -> None:
+        """Hardware version."""
+        return None
+
+    @property
+    def firmware_version(self) -> None:
+        """Firmware version."""
+        return None
+
+    @property
+    def taste(self) -> None:
+        """Taste id."""
+        return self.data["taste"]
+
+    @property
+    def rice(self) -> None:
+        """Rice id."""
+        return self.data["rice"]
+
+    @property
+    def favorite(self) -> None:
+        """Favored recipe id."""
+        return self.data["favs"]
+
+
+class MultiCooker(Device):
+    """Main class representing the multi cooker."""
+
+    _supported_models = [MODEL_MULTI]
+
+    @command(
+        default_output=format_output(
+            "",
+            "Mode: {result.mode}\n"
+            "Menu: {result.menu}\n"
+            "Stage: {result.stage}\n"
+            "Temperature: {result.temperature}\n"
+            "Remaining: {result.remaining}\n"
+            "Cooking delayed: {result.cooking_delayed}\n"
+            "Duration: {result.duration}\n"
+            "Keep warm: {result.keep_warm}\n"
+            "Rice: {result.rice}\n"
+            "Taste: {result.taste}\n"
+            "Favorite: {result.favorite}\n",
+        )
+    )
```

Comment:
```suggestion
    @command()
```
This can be cleaned after adding the necessary decorators :-)

Reply:
done